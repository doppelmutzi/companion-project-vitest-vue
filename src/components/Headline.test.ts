import { mount } from "@vue/test-utils";
import { describe, it, expect } from "vitest";
import Headline from "./Headline.vue";

describe("Headline", () => {
  it("renders correctly", async () => {
    const wrapper = mount(Headline, {
      props: {
        fontColor: "red",
        text: "Hello, World!",
      },
    });

    expect(wrapper.text()).toMatchSnapshot();
  });
});
