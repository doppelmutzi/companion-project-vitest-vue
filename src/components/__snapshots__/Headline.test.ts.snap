// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Headline > renders correctly 1`] = `"Hello, World!"`;
